src/frontend.cpp

```diff
@@ -9,6 +10,7 @@ void Frontend::WriteFile() {
 	GenerateStringStream();
 	fileStream.open(outputFileName);
 	fileStream << stream.rdbuf();
+	chmod(outputFileName.c_str(), S_IRWXU);
```

Comment:
This does not look like something that should be part of this commit.

Reply:
I know i cant get it to go away...